### ISSUE TITLE
Batch folder KMD-OBJ conversion

### ISSUE THREAD

**Issue by Friedslick6** [secaproject/MGS_kmd2obj#2]
As the title reads. It would be a user-friendly addition.
As an appreciator and model archivist I'm glad that someone is working on this again. You've picked a hard game to study haha.

**Comment by devilsnake88** [comment]
Heyyyy ideas's stealer spotted! :p
Secaproject, can I make a video about your tool with my "custom" version (using BATCH file)?

**Comment by secaproject** [comment]
@Friedslick6 I have a non-definitive version of my tool that scan all files on a folder and converts to obj. I want to release when texture works. Meanwhile, I'm working on mgs2 console kmy models, that works on different way as pc.

@devilsnake88 Of course, if you can put my website in the description I will apreciate it. And thanks for your support. Btw, I see your last comment and I'm working on it.

**Comment by devilsnake88** [comment]
Here is the video tutorial with tools:
https://www.youtube.com/watch?v=rD1iFo6OuUw

I had an idea, did you take a look at PSX emulators to see if you can get some interested infos about VRAM?

**Comment by secaproject** [comment]
Yes, but I can't find a relation from vram with the paremeters of kmd file, and I have no answer from neko :(

**Comment by devilsnake88** [comment]
Give me a way to speak with you please like Skype/Steam/etc...

**Comment by secaproject** [comment]
sure, my skype is tbgsergiocalvo ;)
